bin/improver-ensemble-calibration

```diff
@@ -0,0 +1,128 @@
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Script to run ensemble calibration."""
+
+import argparse
+
+import iris
+
+from improver.ensemble_calibration.ensemble_calibration import (
+    EnsembleCalibration)
+from improver.ensemble_copula_coupling.ensemble_copula_coupling import (
+    GeneratePercentilesFromMeanAndVariance, EnsembleReordering)
+
+
+def main():
+    """Load in arguments and get going."""
+    parser = argparse.ArgumentParser(
+    description='Apply the requested ensemble calibration method using '
+                'historical forecast and "truth" data.')
+    parser.add_argument('calibration_method',
+                        metavar='ENSEMBLE_CALIBRATION_METHOD',
+                        choices=['ensemble model output statistics',
+                                 'nonhomogeneous gaussian regression'],
+                        help='The calibration method that will be applied. '
+                             'Supported methods are: "emos" (ensemble model '
+                             'output statistics) and "ngr" (nonhomogeneous '
+                             'gaussian regression).')
+    parser.add_argument('units', metavar='UNITS_TO_CALIBRATE_IN',
+                        help='The unit that calibration should be undertaken '
+                             'in. The current forecast, historical forecast '
+                             'and truth will be converted as required.')
+    parser.add_argument('distribution', metavar='DISTRIBUTION',
+                        choices=['gaussian', 'truncated gaussian'],
+                        help='The distribution that will be used for '
+                             'calibration. This will be dependent upon the '
+                             'input phenomenon. This has to be supported by '
+                             'the minimisation functions in '
+                             'ContinuousRankedProbabilityScoreMinimisers.')
+
+    parser.add_argument('input_filepath', metavar='INPUT_FILE',
+                        help='A path to an input NetCDF file containing the '
+                             'current forecast to be processed.')
+    parser.add_argument('historic_filepath', metavar='HISTORIC_DATA_FILE',
+                        help='A path to an input NetCDF file containing the '
+                             'historic forecast used for calibration.')
```

Comment:
Could you put `historic forecast(s)`? Just to make it clear that usually you'd be expecting multiple historical forecasts e.g. 5-50 days of forecasts.